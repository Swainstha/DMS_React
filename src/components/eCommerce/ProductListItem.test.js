import testSnapFunction from '../../util/testSnapFunction';
import ProductListItem from './ProductListItem';
import React from 'react';


const dummy_data={
    product:{
        thumb:"thumbnail",
        name:"product1",
        price: 75,
        mrp: 80,
        offer:78,
        variant:"black",
        rating:5,
        description:"very good"
    }
}
testSnapFunction("<ProductListItem/>", "Snapshot test for ProductListItem",<ProductListItem {...dummy_data}/>)