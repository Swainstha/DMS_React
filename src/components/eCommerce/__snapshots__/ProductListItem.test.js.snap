// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductListItem/> Snapshot test for ProductListItem 1`] = `ShallowWrapper {}`;
